xml/System.Windows.Diagnostics/ResourceDictionaryDiagnostics.xml

```diff
@@ -154,9 +154,9 @@
         <Parameter Name="uri" Type="System.Uri" />
       </Parameters>
       <Docs>
-        <param name="uri">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="uri">The source URI. The method return <see langword="null" /> if no resource dictionaries were created from <c>uri</c></param>
```

Comment:
I think that "The method return..." should be deleted. The full(er) set of conditions is given in the return value section.